transport/transport-runtime/code_gen_cfg.textproto

```diff
@@ -0,0 +1,7 @@
+vendor_package: "com.google"
+
+include: "com.google.firebase.transport.ClientMetrics"
+include: "com.google.firebase.transport.TimeWindow"
+include: "com.google.firebase.transport.StorageMetrics"
+include: "com.google.firebase.transport.LogSourceMetrics"
+include: "com.google.firebase.transport.LogEventDropped"
```

Comment:
```suggestion
include: "com.google.firebase.transport.ClientMetrics"

```

You only need to specify the "root" proto object, anything it references will automatically be included.